src/styles.css

```diff
@@ -115,6 +128,22 @@ ul li.checked::before {
   background-image: url(./images/checked.png);
 }
 
+ul li span {
+  position: absolute;
+  right: 8px;
+  top: 6px;
+  width: 30px;
+  font-size: 20px;
+  color: #555;
```

Comment:
You should use CSS variables, but it's total up to you: https://www.w3schools.com/css/css3_variables.asp